assets/javascripts/discourse/controllers/admin-plugins-activity-pub-actor-show.js

```diff
@@ -60,27 +63,41 @@ export default class AdminPluginsActivityPubActorShow extends Controller {
 
   @action
   saveActor() {
-    this.saving = true;
+    this.loading = true;
     this.actor.save().then((result) => {
       if (result?.success) {
         updateSiteActor(result.actor);
-
         if (this.actor.isNew) {
-          this.saving = false;
+          this.loading = false;
           return this.router.transitionTo(
             "adminPlugins.activityPub.actorShow",
             result.actor
           );
         }
         this.actor = ActivityPubActor.create(result.actor);
-        this.saveResponse = "success";
-      } else {
-        this.saveResponse = "failed";
       }
-      discourseLater(() => {
-        this.saveResponse = null;
-      }, 3000);
-      this.saving = false;
+      this.loading = false;
+    });
+  }
+
+  @action
+  deleteActor() {
+    this.dialog.yesNoConfirm({
```

Comment:
Can we use a `deleteConfirm` dialog here? We can make `Are you sure you want to delete ACTOR_NAME?` a title, and have the action button use the `danger` color, it's more appropriate for a delete action. 

Also, in the warning message, these two lines are unclear: 

- Posts authored by apb@d1.musaraj.com will be deleted.

Does this refer to ActivityPub posts? I guess so but we should clarify, so that admins don't think that the actor's Discourse posts will get deleted. 

- Other "APB" posts will not be affected.

This is unclear too.

Reply:
Good feedback, thanks. I've made various changes to the delete and destroy confirmation modals along the lines you've identified: https://github.com/angusmcleod/discourse-activity-pub/commit/8f2f194252fe643b05fb8c910cb0a65aa15e5d10